### PR TITLE
DAOS-14965 tests: using correct rd_fac in online_rebuild_single

### PR DESCRIPTION
**Pull request by wangdi1**
Use rd_fac:2 in online_rebuild_single

Test-tags: test_ec_online_rebuild_single
Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: src/tests/ftest/erasurecode/online_rebuild_single.yaml [hunk 1]
```diff
@@ -33,6 +33,9 @@ pool:
   svcn: 1
   control_method: dmg
   pool_query_timeout: 30
+  rf: !mux
+    rf1:
+      properties: rd_fac:2
```
Comment (daltonbohning) [inline src/tests/ftest/erasurecode/online_rebuild_single.yaml:38]: We don't need mux since it's only one variant
```suggestion
  properties: rd_fac:2
```

**Review by daltonbohning — CHANGES_REQUESTED** [review]
The commit pragma should be `Test-tag`, not `Test-tags`
```
Test-tag: test_ec_online_rebuild_single
```

File: src/tests/ftest/erasurecode/online_rebuild_single.yaml (hunk 1 above)
Comment (wangdi1) [reply src/tests/ftest/erasurecode/online_rebuild_single.yaml:38]: ah, thanks! @daltonbohning

**Review by daltonbohning — COMMENTED** [review]
Code LGTM, but seems the test is still failing:
https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-13600/3/tests

**Comment by wangdi1** [comment]
> Code LGTM, but seems the test is still failing: https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-13600/3/tests

ah, the test using nsvc=1 to create the pool, but unfortunately, the killing rank is actually just the PS leader.  I guess since PR-13494 is landed, some previous killing ranks are likely selected as PS leader.  The test should use nsvc = 5, since it will kill 2 ranks.

**Comment by wangdi1** [comment]
the test passed with new parameters, and with test-repeat: 3  @daltonbohning @phender

**Pull request merged**